pyproject.toml

```diff
@@ -67,23 +67,23 @@ python = ">=3.6,<3.9"
 coloredlogs = ">=10,<15"
 sanic = ">=19.12.2,<21.0.0"
 sanic-cors = "^0.10.0"
-requests = "^2.23"
+requests = ">=2.23.0,<2.26.0"
 
 [tool.poetry.dev-dependencies]
 pytest-cov = "^2.10.1"
-coveralls = "^2.1.2"
-pytest = "^5.4.1"
+coveralls = "^2.2.0"
+pytest = "^6.1.2"
 black = "^20.8b1"
 flake8 = "^3.8.2"
 flake8-docstrings = "^1.5.0"
 pytest-sanic = "^1.6.1"
-questionary = "^1.8.1"
+questionary = ">=1.7.0,<1.9.0"
```

Comment:
this is incompatible with what's specified in `rasa`. And why is this a dev dependency in the first place?

Reply:
it's used in the release script.